src/surfing_penguin/API/survey_api.py

```diff
@@ -77,13 +91,17 @@
 class create_survey(Resource):
     @api.marshal_with(api_return_message)
     @api.expect(api_get_survey)
+    @login_required(role='ANY')
```

Comment:
Is the role ANY for create_survey()?

Reply:
Yes, because we just need to check that the user is login, and every user can create survey.

Reply:
OK. I think this would be changed someday but it's fine for this PR.